components/Cards/index.js

```diff
@@ -17,3 +17,70 @@
 // </div>
 //
 // Create a card for each of the articles and add the card to the DOM.
+
+//makes the query selector main div
+const cardsContainer = document.querySelector('.cards-container');
+
+//get the axios
+axios.get('https://lambda-times-backend.herokuapp.com/articles')
+
+  .then((results) => {
+    const bootstrap = results.data.articles.bootstrap;
+    const javascript = results.data.articles.javascript;
+    const jquery = results.data.articles.jquery;
+    const node = results.data.articles.node;
+    const technology = results.data.articles.technology;
+    bootstrap.forEach((obj) => {
+        const create = CreateArticles(obj);
+        cardsContainer.appendChild(create);
+    });
+    javascript.forEach((obj) => {
+        const create = CreateArticles(obj);
+        cardsContainer.appendChild(create);
+    });
+    jquery.forEach((obj) => {
+        const create = CreateArticles(obj);
+        cardsContainer.appendChild(create);
+    });
+    node.forEach((obj) => {
+        const create = CreateArticles(obj);
+        cardsContainer.appendChild(create);
+    });
+    technology.forEach((obj) => {
+        const create = CreateArticles(obj);
+        cardsContainer.appendChild(create);
+    });
+});
+
```

Comment:
You're repeating yourself A LOT on this project. Dedicate some time to refactoring this to make it DRY